src/main/java/com/uber/jenkins/phabricator/coverage/PathResolver.java

```diff
@@ -31,11 +31,11 @@
 import java.util.List;
 import java.util.Map;
 
-public class PathResolver {
+class PathResolver {
```

Comment:
Is this necessary?

Reply:
Does not hurt. Will do another pass of cleaning up access level in a separate pr